### ISSUE TITLE
Error messages when 403, etc. happen

### ISSUE THREAD

**Issue by MaartenLMEM** [dis-moi/extension#801]
For extensions UI sand contributor UIs
Edge cases like 403 (connexion off)

**Comment by christpet** [comment]
To clarify... Is this issue asking for a design to be created for those edge cases? Or do the designs already exist and they need to be implemented? @MaartenLMEM

**Comment by MaartenLMEM** [comment]
Yes UI designs already exist. But, If I remember correctly,  they are quite general. We did not make a list of kind of errors, on which screens, it s just general. Neverthelss, I think I would be an improvment, even if it doesn't precisely fit to each user situation. It's better than no feedback.
@coffeeandpixel knows more about this matter then I do. He can provide 1st designs, etc.

**Comment by christpet** [comment]
Okay thanks, this could be a relatively easy one to implement. I'll put it in the "to do" column for anyone to take when they have a moment.

In the meantime, @coffeeandpixel could you point us to those designs?

**Comment by lutangar** [comment]
I agree with @JalilArfaoui this should be planned. I'm seeing this has been moved to the [Performance board](https://github.com/orgs/dis-moi/projects/12) but should we move this to the extension todo as well @MaartenLMEM ?

**Comment by MaartenLMEM** [comment]
I agree, I am just waiting for the design.
